package.json

```diff
@@ -68,8 +68,9 @@
     "@veramo/remote-server": "4.2.0",
     "@veramo/selective-disclosure": "4.2.0",
     "@veramo/url-handler": "4.2.0",
+    "typeorm": "0.3.10",
```

Comment:
If you are doing a global resolution, why are you changing the the values in the subpackages, that are not really changing much of the behaviour. Basically Veramo has ^0.3.10. Without this resolution the end result of any implementation still is  0.3.12, given that version satisfies both the original Veramo version range as well as ours.

In other words the other changes do nothing

Reply:
This is the code from the root package.json:

```json
"resolutions": {
    "@veramo/cli": "4.2.0",
    "@veramo/core": "4.2.0",
    "@veramo/credential-w3c": "4.2.0",
    "@veramo/data-store": "4.2.0",
    "@veramo/did-jwt": "4.2.0",
    "@veramo/credential-ld": "4.2.0",
    "@veramo/utils": "4.2.0",
    "@veramo/did-manager": "4.2.0",
    "@veramo/kms-local": "4.2.0",
    "@veramo/key-manager": "4.2.0",
    "@veramo/credential-eip712": "4.2.0",
    "@veramo/did-comm": "4.2.0",
    "@veramo/message-handler": "4.2.0",
    "@veramo/did-discovery": "4.2.0",
    "@veramo/did-provider-ethr": "4.2.0",
    "@veramo/did-provider-key": "4.2.0",
    "@veramo/did-provider-web": "4.2.0",
    "@veramo/did-resolver": "4.2.0",
    "@veramo/remote-client": "4.2.0",
    "@veramo/remote-server": "4.2.0",
    "@veramo/selective-disclosure": "4.2.0",
    "@veramo/url-handler": "4.2.0",
    "typeorm": "0.3.10",
    "**/@mattrglobal/jsonld-signatures-bbs/jsonld": "4.0.1",
    "**/jsonld": "link:./node_modules/@digitalcredentials/jsonld",
    "**/isomorphic-webcrypto": "link:./node_modules/@sphereon/isomorphic-webcrypto"
  }
```
as you can see  *"typeorm": "0.3.10"* is already included in the resolutions

Reply:
Okay then the resolution value is wrong or something else is up.

Veramo is using ^0.3.10 and we were using ^0.3.12. Since 0.3.12 is the latest it should resolve to 0.3.12 for both.

In other words, you should find out which package apparently is causing a hard fix on another version, as we are now just removing symptomps without knowing the actual cause